### PR TITLE
[MAIN] [STRATCONN] 4146 Facebook Anon_id and MadId added in Action Field

### PR DESCRIPTION
**Pull request by AnkitSegment**
_A summary of your pull request, including the what change you're making and why._

Tested successfully in stage, the parameter is sent to FB and accepted:

Screenshot 2024-10-03 at 12 22 23 PM

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [X] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [If destination is already live] Tested for backward compatibility of destination. **Note:** New required fields are a breaking change.
- [X] [Segmenters] Tested in the staging environment
- [ ] [Segmenters] [If applicable for this change] Tested for regression with Hadron.

### CODE REVIEW

**Review by itsarijitray — DISMISSED** [review]

**Comment by Innovative-GauravKochar** [comment]
@AnkitSegment Changes looks good, Can you please add summary of your pull request, including the what change you're making and why ?
Can you please add evidence when you don't send either of them or both of them ?

File: packages/destination-actions/src/destinations/facebook-conversions-api/fb-capi-user-data.ts [hunk 1]
```diff
@@ -113,6 +113,12 @@ export const user_data_field: InputField = {
         'This field represents unique application installation instances. Note: This parameter is for app events only.',
       type: 'string'
     },
+    madId: {
+      label: 'advertiser ID (madid)',
```
Comment (nick-Ag) [inline packages/destination-actions/src/destinations/facebook-conversions-api/fb-capi-user-data.ts:117]: Nit
```suggestion
      label: 'Advertiser ID (madid)',
```

File: packages/destination-actions/src/destinations/facebook-conversions-api/fb-capi-app-data.ts [hunk 2]
```diff
@@ -99,6 +101,18 @@ export const app_data_field: InputField = {
...
       type: 'string'
     },
+    anonId: {
+      label: 'Install ID (anon_id)',
+      description:
+        'This field represents unique application installation instances. Note: This parameter is for app events only.',
+      type: 'string'
+    },
+    madId: {
+      label: 'advertiser ID (madid)',
```
Comment (nick-Ag) [inline packages/destination-actions/src/destinations/facebook-conversions-api/fb-capi-app-data.ts:111]: Nit
```suggestion
      label: 'Advertiser ID (madid)',
```

**Review by nick-Ag — CHANGES_REQUESTED** [review]
Left a couple of nitpicks. Code looks good otherwise. I agree with @Innovative-GauravKochar that we should get some testing evidence in the cases where neither are sent or only one of each are sent

**Review by nick-Ag — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
hi @AnkitSegment PR deployed

**Pull request merged**